api/filters.go

```diff
@@ -530,6 +531,58 @@ func (api *FilterAPI) removeFilterBlueprintDimensionOption(w http.ResponseWriter
 	log.Info("delete filtered blueprint", log.Data{"filter_blueprint_id": filterID, "dimension": name})
 }
 
+func (api *FilterAPI) getFilterOutputPreview(w http.ResponseWriter, r *http.Request) {
```

Comment:
Consider adding test coverage to this function - ensuring that the expected limit is passed down when specified and the default is applied when its not.

Reply:
done